app/scripts/lib/ComposableObservableStore.js

```diff
@@ -40,7 +40,9 @@ class ComposableObservableStore extends ObservableStore {
   getFlatState () {
     let flatState = {}
     for (const key in this.config) {
-      flatState = { ...flatState, ...this.config[key].getState() }
+      const controller = this.config[key]
+      const state = controller.getState ? controller.getState() : controller.state
```

Comment:
This is a temporary patch to `ComposableObservableStore` to provide GABA API compatibility. Once we're fully on GABA, this entire module goes away and we'll be using `gaba/ComposableController` instead.